file-formats/uist/examples/z_aff_example_uist.uist.json

```diff
@@ -0,0 +1,18 @@
+{
```

Comment:
It seems to me that this is an issue in our validation for JSON examples. Do I miss anything?

Reply:
there is https://github.com/python-jsonschema/jsonschema/issues/1039, but I have no idea, its Python...

Reply:
I have just created an issue #498

Reply:
@ViktoriaFreidel  JFYI. Please ignore the error for now. I will also ask you to remove the multiple of value from your ABAP type

Reply:
Ok. No problem. Today I will send update taday.